src/waddle/argparse.py

```diff
@@ -124,6 +121,12 @@ def create_waddle_parser():
         action="store_true",
         help="Skip converting audio files to WAV format.",
     )
+    preprocess_parser.add_argument(
```

Comment:
should the default be to transcribe? I wonder how much transcription would be useful if you're editing locally without ui

Reply:
Even without a UI, I was referring to the transcript when editing, so I set it as the default. However, if we’re building a UI, I think most people who go out of their way to run it from the CLI probably don’t need the transcript. So, I’ll set the default to `transcription_path = None` and disable transcription by default.